agent/engine/task_manager_linux.go

```diff
@@ -0,0 +1,22 @@
+//go:build linux
```

Comment:
like other os specific logic in the agent repository, we need a _unsupported.go file for this

Reply:
Added this in [92b4941](https://github.com/aws/amazon-ecs-agent/pull/4326/commits/92b4941fe4a9b7eb6480f6eb9cf4c18565dd99d9)